extensions/amp-list/amp-list.md

```diff
@@ -220,6 +220,23 @@ In several cases, we may need the `<amp-list>` to resize on user interaction. Fo
 </amp-list>
 ```
 
+### Initialization from amp-state
+
+In some cases, it may be desirable to have your `<amp-list>` component initialize off of `<amp-state>` rather than from a json endpoint.
+You may do that by utilizing the `amp-state:` protocol in the `src` attribute. The data in state must follow the same rules as the json that
```

Comment:
Add that this requires including `amp-bind` extension.

Reply:
Done